LondonBritishLibrary/orient/BLorient11764.xml

```diff
@@ -0,0 +1,155 @@
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/>ጸሎት፡ በእንተ፡ ሌጌዎን፡ ርኩስ፡ ዘይሰልብ፡ ልበ፡ ሰብእ፡ ወያጸልም፡ አዕይንተ፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <title ref="LIT7063Devils">Prayer catching devils</title>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/>አላሁማ፡ ወያኑራ፡ ሐሽም፡ <gap reason="ellipsis"/> ዘነበቦሙ፡ 
+                                ለአጋ<surplus>ጋ</surplus>ንንት፡ ሎፍሐም፡ ከማሁ፡ እስሮሙ፡ ለቡዳ፡ መሰርይ፡ ወሌጌዎን፡ ዕኩይ፡ ለእመ፡ አኀዞ፡ እደ፡ ሰብእ፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i5">
+                            <title ref="LIT7064PrayerChest">Prayer against bāryā, chest pain (wǝgʿat), diseases, and charms</title>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ዕልመይከክን፡ ረቢ፡ ታኦስ፡ ባኦስ፡ <gap reason="ellipsis"/> ከማሁ፡ ዓፅርዕ፡ ወአዕትት፡ 
+                                ሕማመ፡ ባርያ፡ ወውግዓት፡ ወደዌ፡ ወሥራይ፡ እምላዕለ፡ ነፍሱ፡ ወሥጋሁ፡ ለገብረ፡ እግዚአብሔር፡ </incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i6">
+                            <title ref="LIT7045Colic"/>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ጸሎተ፡ ቍረፀት፡ ኄር፡ ኪሳ፡ ኪሳ፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Scroll">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">1</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>1730</height>
+                                        <width>70</width>
+                                    </dimensions>
+                                    <note>Composed of three strips.</note>
+                                </extent>
+                            </supportDesc> 
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>Square, rather careful script.</desc>
+                                <date notBefore="1600" notAfter="1799" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
+                                <desc>Coloured magic picture.</desc>
```

Comment:
you can mark up magic as a term, then one could search for all drawings containing the keyword in our filtered search